starter-code/index.js

```diff
@@ -1,10 +1,45 @@
-var SortedList = function() {};
-
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+var SortedList = function() {
+  this.items = [],
```

Comment:
Mistake was made here... Object LITERALS end with commas on each line BUT Object Constructors end with Semicolons. You added commas 🧐